lesson_2/test1.py

```diff
@@ -0,0 +1,13 @@
+
+# 1. a function that returns the sum of two numbers
+
+# START
+# GET 2 numbers from user
```

Comment:
Here's how I would organize this code:

```suggestion
def sum(num1, num2):
    return num1 + num2

num1 = int(input('Enter a number: '))
num2 = int(input('Enter another number: '))

print(sum(num1, num2))
```

Reply:
fixed it but why have have num1 and num2 intialized below? is it bc we are calling function below so its just cleaner to have it above  the call? (ie line 9 )//also I wont leave comments like that in future ik its ugly but wanted to try it out so it code be pseudocode step and then matching code